src/freenas/usr/local/libexec/freenas-debug/active_directory/active_directory.sh

```diff
@@ -212,14 +212,14 @@ __EOF__
 	#
 	section_header "Active Directory Users and Groups"
 	section_header "Users - 'wbinfo -u'"
-	wbinfo -u
+	wbinfo -u | head -200
 	section_header "Groups - 'wbinfo -g'"
-	wbinfo -g
+	wbinfo -g | head -200
 	section_header "Using getent"
 	section_header "Users - 'getent passwd'"
-	getent passwd
+	getent passwd | head -200
 	section_header "Groups - 'getent group'"
-	getent group
+	getent group | head -200
```

Comment:
Since we're limiting the output of the users/groups it would be nice to have another section that provides the quanity of users/groups - this is a helpful information and sometimes explains a lot.